### ISSUE TITLE
Elevated circulating S-adenosyl-L-homocysteine concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8153]
**Preferred term label:**

Elevated circulating S-adenosyl-L-homocysteine concentration
**Synonyms**
Elevated AdoHcy concentration

**Definition (free text, please give PubMed ID)**
Increased concentration of S-adenosyl-L-homocysteine in the blood circulation.

**Parent term (use hpo.jax.org/app)**

Abnormal circulating non-proteinogenic amino acid concentration HP:0033109

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614300 
PMID: 21963049   5/6

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
